controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -38,6 +38,7 @@ import (
 	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1alpha3"
 	"sigs.k8s.io/cluster-api/util"
 	"sigs.k8s.io/cluster-api/util/certs"
+	imageutil "sigs.k8s.io/cluster-api/util/container"
```

Comment:
Why not use `container` like everywhere else?

Reply:
it was conflicting with a actual container in the same func

Reply:
Should probably be `containerutil`, wdyt?

Reply:
sure, will use `containerutil `